repo/packages/A/arangodb/1/mantl.json

```diff
@@ -0,0 +1,13 @@
+{
+  "mesos": {
+    "master": "zk://zookeeper.service.consul:2181/mesos"
+  },
+  "arangodb": {
+    "id": "arangodb",
+    "volume-path": "/data/arangodb",
+    "mesos-authenticate":"{{mantl.mesos.authentication-enabled}}"
+    {{#mantl.mesos.authentication-enabled}},
+    "principal":"{{mantl.mesos.principal}}",
+    "secret": "{{mantl.mesos.secret}}"{{/mant.mesos.authentication-enabled}}
```

Comment:
small type here: `mant.mesos.authentication-enabled` should be `mantl.mesos.authentication-enabled`